adapters/yeahmobi/yeahmobi.go

```diff
@@ -187,3 +187,10 @@ func getBidType(impId string, imps []openrtb2.Imp) openrtb_ext.BidType {
 	}
 	return bidType
 }
+
+type yeahmobiBidExt struct {
+	VideoCreativeInfo *yeahmobiBidExtVideo `json:"video,omitempty"`
+}
+type yeahmobiBidExtVideo struct {
+	Duration *int `json:"duration,omitempty"`
+}
```

Comment:
Requesting you to please remove these structs as well. There should ideally be no change to yeanmobi.go file

Reply:
ok，i have removed these codes